pkgs/tools/games/gog-downloader/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, makeWrapper, lib, php }:
+
+let
+  pname = "gog-downloader";
+  version = "1.2.3";
+  hash = "0kv09vmrz6dcjvaf2sw89cpb14wdr01r1kkzg84szfxwzr5bcc9p";
+in
+stdenv.mkDerivation {
+  inherit pname version hash;
+
+  src = fetchurl {
+    url = "https://github.com/RikudouSage/GogDownloader/releases/download/v${version}/gog-downloader";
```

Comment:
```suggestion
    url = "https://github.com/RikudouSage/GogDownloader/releases/download/v${finalAttrs.version}/gog-downloader";
```

Reply:
Do you have a link or anything (like a quick summary) where I can learn the difference? In the few packages I've seen I haven't noticed `finalAttrs` and I'd like to learn!

Reply:
See my comment in here: https://github.com/NixOS/nixpkgs/pull/245971#discussion_r1278619783